flatpak_builder_lint/appstream.py

```diff
@@ -12,8 +12,33 @@
 def validate(path: str) -> dict:
     if not os.path.isfile(path):
         raise FileNotFoundError("AppStream file not found")
+
+    overrides = {
+        "all-categories-ignored": "error",
+        "category-invalid": "error",
+        "cid-desktopapp-is-not-rdns": "error",
+        "cid-has-number-prefix": "error",
+        "cid-maybe-not-rdns": "error",
+        "cid-missing-affiliation-gnome": "error",
+        "cid-rnds-contains-hyphen": "error",
+        "content-rating-missing": "error",
+        "desktop-app-launchable-omitted": "error",
+        "desktop-file-not-found": "error",
+        "developer-id-missing": "error",
+        "invalid-child-tag-name": "error",
+        "metainfo-filename-cid-mismatch": "error",
+        "metainfo-legacy-path": "error",
+        "metainfo-multiple-components": "error",
+        "name-has-dot-suffix": "error",
```

Comment:
The summary one is also desired right https://github.com/flathub/flathub/issues/4635 ?

Reply:
Not super sure. Dot suffix in app name looks immediately wrong, a dot in summary shouldn't cause build errors imho.

Reply:
Ok I guess I'll add it as a warning then, dots in summary are caught during review manually.